scripts/deploy.sh

```diff
@@ -126,13 +126,14 @@ rebuild_regs() {
 }
 
 update_docs() {
-  documents_folder=WcaOnRails/public/documents
+  public_dir=WcaOnRails/public
   tmp_dir=/tmp/wca-documents-clone
 
   rm -rf $tmp_dir
   git clone --depth=1 --branch=build https://github.com/thewca/wca-documents.git $tmp_dir
-  rm -rf $documents_folder
-  mv $tmp_dir $documents_folder
+  rm -rf $public_dir/documents
+  rm -rf $public_dir/edudoc
+  mv $tmp_dir/documents $tmp_dir/edudoc $public_dir
```

Comment:
Wow, I never knew `mv` could take multiple arguments in the first position! :heart_eyes: